pythonwhois/net.py

```diff
@@ -91,4 +91,10 @@ def whois_request(domain, server, port=43):
 		if len(data) == 0:
 			break
 		buff += data
-	return buff.decode("utf-8")
+	encodings = ("utf-8", "iso-8859-1")
+	for encoding in encodings: # This should probably not be a permanent solution.
```

Comment:
No need for that comment IMO. This is a perfectly fine solution. The RFC does not specify an encoding (rather comments on it as an issue), so the original assumption that it's valid UTF-8 data is where the problem lies.